db/seeds.rb

```diff
@@ -10,8 +10,7 @@
     User.find_or_create_by!(email: email) do |user|
       user.skip_confirmation!
       user.reset_password('password', 'password')
-      user.unconfirmed_mobile = format('+1 (415) 555-01%02d', index)
-      user.mobile_confirm
+      user.mobile_confirm(format('+1 (415) 555-01%02d', index))
```

Comment:
How about:
```ruby
user.mobile = format('+1 (415) 555-01%02d', index)
user.mobile_confirmed_at = Time.current
```
2 reasons:

- Since seeds are meant to create users and set attributes, it's best to stick with attribute assignment as opposed to relying on methods that might change or might be removed.
- The `mobile_confirm` method doesn't belong in the User model